packages/node_modules/nav-frontend-knapper/src/flatknapp.js

```diff
@@ -0,0 +1,10 @@
+import React, { Component } from 'react';
+import KnappBase from './index';
+
+class Flatknapp extends Component { // eslint-disable-line react/prefer-stateless-function
```

Comment:
Kanskje komponenten burde hete FlatKnapp istedet for Flatknapp?
Er det en "flat knapp" eller en "flatknapp"?